src/js/components/LogItem/utils/createLogTitle.jsx

```diff
@@ -61,8 +63,8 @@ const createLogTitle = (log: ILogItem): ReactElement => {
                             <HighlightedText>{log.data.commands.length}</HighlightedText>
                             {'; '}
                             {
-                                hasLrc(log.commandType) &&
-                                log.data.lrc.expected !== log.data.lrc.actual
+                                hasLrc(log.commandType)
```

Comment:
such simple cases it's possible to write in a single line

Reply:
updated as suggested